src/ontodia/diagram/paperArea.tsx

```diff
@@ -274,6 +276,12 @@ export class PaperArea extends React.Component<Props, State> {
             }
         }
 
+        if (isIE11()) {
```

Comment:
It would be great to move this polyfill to `polyfills.ts` as well. (Also don't forget to remove `isIE11` import from this file.)